borg/remote.py

```diff
@@ -287,6 +290,8 @@ def fetch_from_cache(args):
                         type, msgid, error, res = unpacked
                         if msgid in self.ignore_responses:
                             self.ignore_responses.remove(msgid)
+                            if error:
+                                handle_error(error, res)
```

Comment:
The way I understand it had a similar impact as the original issue. An error (during e.g. a put()) would still be visible in the log, but not cause a program abort and may have stored/corrupted data.

Re. changelog I think it's the same bug in terms of description, that was caused two different ways for local and remote repos each.